pkg/kratos/handlers.go

```diff
@@ -132,7 +133,16 @@ func (a *API) handleCreateFlow(w http.ResponseWriter, r *http.Request) {
 			a.cookieManager.ClearStateCookie(w)
 		}
 	} else {
-		response, cookies, err = a.handleCreateFlowNewSession(r, aal, returnTo, loginChallenge, refresh)
+		responseFlow, cookies, err = a.handleCreateFlowNewSession(r, aal, returnTo, loginChallenge, refresh)
+		setCookies(w, cookies)
```

Comment:
Cookies are set twice now, I don't remember if this is browser specific behavior, handled by golang or we can expect all browsers to simply ignore the duplicate cookies, but I think we should not do that.